model/modelindexer/indexer.go

```diff
@@ -340,6 +344,8 @@ func (i *Indexer) flushActive(ctx context.Context) error {
 	err := i.flush(ctx, bulkIndexer)
 	bulkIndexer.Reset()
 	i.available <- bulkIndexer
+	// i.mu.RLock currently held
```

Comment:
did you mean to leave this comment?

Reply:
I did, but if you think it's unnecessary, happy to remove

Reply:
I was confused because we're just doing an atomic operation on the pointer of that variable, so I was curious to know why that's pointed out. We're not taking the read lock when the `.Stats()` method is called, so I'm curious why you added the comment.